pkg/reconciler/shared/tektonconfig/upgrade/post_upgrade.go

```diff
@@ -70,3 +80,66 @@ func upgradeStorageVersion(ctx context.Context, logger *zap.SugaredLogger, k8sCl
 
 	return nil
 }
+
+// Remove the all  cluster tasks installer sets
+func removeAllClusterTasksInstallerSets(ctx context.Context, logger *zap.SugaredLogger, k8sClient kubernetes.Interface, operatorClient versioned.Interface, restConfig *rest.Config) error {
+
+	if !v1alpha1.IsOpenShiftPlatform() {
+		return nil
+	}
+
+	tisClient := operatorClient.OperatorV1alpha1().TektonInstallerSets()
+
+	ctLS := v1.LabelSelector{
```

Comment:
Should we use `clusterTaskLabelSelector`  ?

Reply:
yes, I think and looks more declarative